### PR TITLE
[NEW][ADD][8.0][pos_payment_term] Feature/pos payment terms

### PR DESCRIPTION
**Pull request by mileo**
https://github.com/OCA/pos/issues/104

### CODE REVIEW

File: pos_payment_terms/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+###############################################################################
+#     Point Of Sale - Payment Terms for POS Odoo
+#    Copyright (C) 2016 KMEE INFORMATICA LTDA (http://www.kmee.com.br)
```
Comment (mileo) [inline pos_payment_terms/__openerp__.py]: @lfdivino 
Please use small copyright like this one: https://github.com/OCA/bank-payment/blob/8.0/account_banking_payment_export/__openerp__.py#L2

File: pos_payment_terms/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from . import point_of_sale
```
Comment (mileo) [inline pos_payment_terms/models/__init__.py]: New line

File: pos_payment_terms/views/pos_template.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+<template id="pos_payment_terms_assets"
+          name="pos_payment_terms_css"
+          inherit_id="point_of_sale.index">
+    <xpath expr="//script[@id='loading-script']" position="before">
+        <link rel="stylesheet" href="/pos_payment_terms/static/src/css/pos.css"/>
+    </xpath>
+</template>
+
+</data>
+</openerp>
```
Comment (mileo) [inline pos_payment_terms/views/pos_template.xml]: New line

**Comment by pedrobaeza** [comment]
Please call it in singular: `pos_payment_term`

File: pos_payment_terms/static/src/xml/pos.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+                <option value="">Nenhuma</option>
+                <t t-foreach='widget.pos.payment_terms' t-as='payment_term'>
+                    <option t-att-value='payment_term.id'>
+                        <t t-esc='payment_term.name'/>
+                    </option>
+                </t>
+            </select>
+        </t>
+    </t>
+</templates>
```
Comment (mileo) [inline pos_payment_terms/static/src/xml/pos.xml]: New line

**Comment by mileo** [comment]
Please fix travis errors: https://travis-ci.org/OCA/pos/jobs/134097131

**Comment by mileo** [comment]
Hi, @pedrobaeza Could you please label as needs review?

When you create a pos.order with payment_term with multiple lines, pos order create a statement with multiple lines too:

captura de tela de 2016-06-21 00 35 20

Therefore statement create multiple move lines:

captura de tela de 2016-06-21 01 42 13

But until now the due dates are not correct. Me and the @lfdivino held a stand up meeting to analyze the best solution in this case: The Odoo core breaks very badly the procedures involved in the creation of journal entries and often without passing the context. So that the solution can be extended data models involved adding due_date and payment_term in bank.statement.line.

If anyone has any suggestions about it would be very helpfull. Thanks in advance.

**Comment by pedrobaeza** [comment]
Done

**Comment by rafaelbn** [comment]
Hi @mileo , please could you rebase? We have to see if travis gets green with last changes. Thanks

**Comment by legalsylvain** [comment]
ping @mileo

**Comment by rafaelbn** [comment]
Closing by old without any answer

This module is quite interesting I'm :cry: but travis is :red_circle:

**Pull request closed without merging**